resources/docs/erp_fhirversion_changes-source.adoc

```diff
@@ -1,5 +1,15 @@
 = E-Rezept FHIR-Package Änderungen bei Versionsübergängen image:gematik_logo.png[width=150, float="right"]
-include::./config-source.adoc[]
+// asciidoc settings for DE (German)
+// ==================================
+:imagesdir: ../images
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:toc: macro
+:toclevels: 3
+:toc-title: Inhaltsverzeichnis
```

Comment:
Das hier ist nicht gut. In der Source sollte man nur die kurze Version nutzen, also include::./config-source.adoc[]